<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deluser</class>
 <widget class="QDialog" name="deluser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>14</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="my_system.qrc">
    <normaloff>:/my_system/Resource/ico.png</normaloff>:/my_system/Resource/ico.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请输入工号:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="delID_lineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="delfind_pushButton">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>用户信息如下:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ID_label">
     <property name="text">
      <string>工号:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="type_label">
     <property name="text">
      <string>角色:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="DEL_pushButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="exit_pushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="my_system.qrc"/>
 </resources>
 <connections/>
</ui>
